xbmc/addons/AddonDatabase.cpp

```diff
@@ -233,6 +234,15 @@ void CAddonDatabase::UpdateTables(int version)
     m_pDS->exec("UPDATE installed SET origin = addonID WHERE (origin='') AND "
                 "EXISTS (SELECT * FROM repo WHERE repo.addonID = installed.addonID)");
   }
+  if (version < 32)
+  {
+    m_pDS->exec(
+        "CREATE TABLE update_rules (id integer primary key, addonID text, updateRule INTEGER)");
+    m_pDS->exec("INSERT INTO update_rules (addonID, updateRule) SELECT addonID, 1 updateRule FROM "
```

Comment:
Is the `1` a typo here?

Reply:
no 1 = `user_disabled_auto_update` the default value in the new table if there are datasets in the blacklist-table

Reply:
Cool